members/FUTS344a83b6.yaml

```diff
@@ -54,52 +59,54 @@ contact_form:
           Veterans: Veterans
     - fill_in:
       - name: First name
-        selector: "#edit-first-name"
+        selector: "#form-field-first_name"
         value: $NAME_FIRST
         required: true
       - name: Last name
-        selector: "#edit-last-name"
+        selector: "#form-field-last_name"
         value: $NAME_LAST
         required: true
       - name: Street
-        selector: "#edit-address-1"
+        selector: "#form-field-address_1"
         value: $ADDRESS_STREET
         required: true
       - name: Street2
-        selector: "#edit-address-2"
+        selector: "#form-field-address_2"
         value: $ADDRESS_STREET_2
         required: false
       - name: City
-        selector: "#edit-city"
+        selector: "#form-field-city"
         value: $ADDRESS_CITY
         required: true
       - name: zip
-        selector: "#edit-zip"
+        selector: "#form-field-zip_code"
         value: $ADDRESS_ZIP5
         required: true
       - name: phone
-        selector: "#edit-phone"
+        selector: "#form-field-phone_number"
         value: $PHONE
-        required: false
+        required: true
       - name: email
-        selector: "#edit-email"
+        selector: "#form-field-email_address"
         value: $EMAIL
         required: true
       - name: Subject
-        selector: "#edit-subject"
+        selector: "#form-field-subject"
         value: $SUBJECT
         required: true
       - name: Message
-        selector: "#edit-message"
+        selector: "#form-field-message"
         value: $MESSAGE
         required: true
     - click_on:
       - value: Submit
-        selector: button#edit-actions-submit
+        selector: div.elementor-col-100.e-form__buttons button
```

Comment:
`section#main-box div.elementor-field-type-submit.e-form__buttons button` would be better here because 1) you're not using a style class as part of the selector in case that changes and 2) there is another `div.e-form__buttons` on the page so it'll be good to distinguish between the two

Reply:
Updated. I looked in chrome devtools and confirmed that both `div.elementor-col-100.e-form__buttons button` and `section#main-box div.elementor-field-type-submit.e-form__buttons button` are unique. But I agree the latter one is more specific.